analysis/study_definition.py

```diff
@@ -102,9 +147,5 @@
         },
     ),
 
-    # comorbidities
-    has_copd = patients.with_these_clinical_events(
-        copd_codes,
-        between = ["index_date", "index_date + 3 months"]
-    )
+    **loop_over_codes(antibacterial_codes),
```

Comment:
This will be creating >2500 columns which I don't think will be able to be handled. I think initially you might have to settle for episodes potentially containing multiple antibacterials.

Reply:
for now I've removed loop_over_codes